web/olga/analytics/models.py

```diff
@@ -76,6 +76,19 @@ class InstallationStatistics(models.Model):
     )
 
     @classmethod
+    def get_stats_for_this_day(cls, edx_installation_object=None):
+        """
+        Provide model, for given installation object, that was created today, or None if it not exist.
```

Comment:
Please, an empty line between doc's summary line and another part of the doc strings.